plugin/src/compilation/decompiler.ml

```diff
@@ -17,26 +17,41 @@ open Utilities
 open Zooming
 open Nameutils
    
+open Ltac_plugin
+open Tacarg
+   
 (*
  * Only in OCaml 4.08.0 onward, so we implement ourselves
  *)
 let filter_map f l =
   let f_somes = List.filter (fun o -> Option.has_some o) (List.map f l) in
   List.map Option.get f_somes
+
+(* Drop the first n elements off a list *)
+let rec drop n xs =
```

Comment:
Is this in the OCaml standard library? I would be surprised if not. Or like filter_map is it only in newer versions of OCaml?

Reply:
Unfortunately not :/ https://caml.inria.fr/pub/docs/manual-ocaml/libref/List.html
I'll put this and `filter_map` in coq-plugin-lib with `take`.